_posts/en/pages/2016-01-18-segwit-wallet-dev.md

```diff
@@ -138,12 +132,12 @@ If a wallet supports script types other than just single signature, such as mult
     * The last witness item of the corresponding witness field MUST be the <code>witnessScript</code>
     * The new [BIP143][] signature generation algorithm is applied:
         * Without using any OP_CODESEPARATOR, the <code>scriptCode</code> is <code>witnessScript</code> preceeded by a <code>compactSize</code> integer for the size of <code>witnessScript</code>. For example, if the script is OP_1 (<code>0x51</code>), the <code>scriptCode</code> being serialized is (<code>0x0151</code>)
-        * For any unusal scripts containing OP_CODESEPARATOR, please refer to [BIP143][] for the exact semantics
+        * For any unusual scripts containing OP_CODESEPARATOR, please refer to [BIP143][] for the exact semantics
     * Any witness stack items before the <code>witnessScript</code> are used as the input stack for script evaluation. The input stack is not interpreted as script. For example, there is no need to use a <code>0x4c</code> (OP_PUSHDATA1) to "push" a big item.
     * To verify the correctness of signature generation and stack serialization, please always test against the examples in [BIP143][]
     * [Example](http://n.bitcoin.ninja/checktx?txid=954f43dbb30ad8024981c07d1f5eb6c9fd461e2cf1760dd1283f052af746fc88)
 
-### Advanced designs
+#### Segwit native addresses (optional) {#advanced-designs}
```

Comment:
Whoops, this seems to have reduced the header level by one.  Otherwise, this update looks good, thanks!